### ISSUE TITLE
Color contrast fails on #4171C4 and #E8E8E8

### ISSUE THREAD

**Issue by spenoff** [dequelabs/axe-core#4983]
### Product

axe-core

### Product Version

4.118.0

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
When checking color contrast on with #4171C4 and #E8E8E8, the color contrast should pass because it meets the 3:1 requirement

#### Actual
color contrast fails

#### How to Reproduce
https://codesandbox.io/p/sandbox/f7hvgn

#### Additional context
Any thing else we should know about the issue?
